lib/tasks/dev.rake

```diff
@@ -63,6 +63,13 @@ end
               acronym: "BCH",
               image_url: "https://s2.coinmarketcap.com/static/img/coins/200x200/1831.png",
               mining_type: MiningType.all.sample
+              },
+
+              {
```

Comment:
unrelated ffrom this PR but I would recommend looking into [rubocop](https://github.com/rubocop/rubocop) this will handle stlying your repo consistently which makes the code easier to read

Reply:
Thanks for the suggestion, will take a loot at it